pkg/piecestore/rpc/server/psdb/psdb.go

```diff
@@ -79,6 +96,11 @@ func Open(ctx context.Context, DataPath, DBPath string) (db *DB, err error) {
 		return nil, err
 	}
 
+	_, err = tx.Exec("CREATE TABLE IF NOT EXISTS `bwusagetbl` (`size` INT(10), `daystartdate` INT(10), `dayenddate` INT(10));")
```

Comment:
Use timestamps instead of INT

Reply:
Egon I added the new functions to be inline with existing functions. I will bring it up in stand up and see what Alex thoughts on this. Will keep you updated on this.

Reply:
we used INTs because it's just easier to interact with(Sort and select ranges) unix timestamps